ui/plugins/auto_classification/main.html

```diff
@@ -39,11 +39,13 @@
           <select
              ng-if="line.type === 'structured' && line.updateText.length > 1"
              ng-model="line.updateSelectOption"
+             title="{{loggedIn() ? '' : 'Must be logged in'}}"
+             ng-disabled="!loggedIn"
```

Comment:
You need to call loggedIn() as a function, not just get the value of the variable (which is always true).